src-server/server.ts

```diff
@@ -0,0 +1,18 @@
+import fastify from 'fastify';
+import cors from '@fastify/cors'
+import data from './data/*';
+
+const api = fastify();
+api.register(cors)
+
+api.get('/', (req, res) => {
+  return 'Hello world!';
+})
+
+for (const file in data) {
```

Comment:
This is probably going to change to something more robust eventually. I was thinking of migrating all the scraping stuff into the Node.js backend so we'd have a single codebase for everything; does that seem reasonable?